code/midtown/mmdyna/bndtmpl.cpp

```diff
@@ -20,6 +20,74 @@ define_dummy_symbol(mmdyna_bndtmpl);
 
 #include "bndtmpl.h"
 
+#include "vector7/geomath.h"
+
+#include "bndtmpl2.h"
+#include "poly.h"
+
+void mmBoundTemplate::ComputeBounds()
+{
+    GetBoundInfo(NumVerts, Verts, &BBMin, &BBMax, &Center, &Radius);
+    RadiusSqr = Radius * Radius;
+}
+
+void mmBoundTemplate::ComputeEdges()
+{
+    i32 total_verts = 0;
+
+    for (i32 i = 0; i < NumPolys; ++i)
+    {
+        total_verts += Polygons[i].GetNumVerts();
+    }
+
+    i32* edge_1s = new i32[4 * total_verts];
+    i32* edge_2s = new i32[4 * total_verts];
+
+    i32 num_edges = 0;
+
+    for (i32 j = 0; j < NumPolys; ++j)
+    {
+        mmPolygon& poly = Polygons[j + 1];
+
+        i32 num_verts = poly.GetNumVerts();
+
+        i32 v1 = poly.MtlIndex + num_verts; // ?
+        // Original line: v1 = *((__int16 *)&this->Polygons[j + 1].MtlIndex + num_verts);
```

Comment:
It's doing `poly.VertIndices[num_verts - 1]` (MtlIndex is stored 2 bytes before the VertIndices)